recipes/mp2d/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "mp2d" %}
+{% set version = "1.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Chandemonium/MP2D/archive/c6c7f866d4c1913ff38854b2c38450184888798a.zip
+  sha256: 3de7b882472fb313d3754ceae3439293a2e1cde63a3e9511ad659d4bf29fd076
+
+build:
+  binary_relocation: true
+  skip: true  # [win]
+  number: 0
+
+requirements:
+  build:
+    - cmake >=3.5
+    - {{ compiler('cxx') }}
+    - make  # [not win]
+
+test:
+  commands:
+    - test -f $PREFIX/share/mp2d/mp2d_parameters.dat
+    - mp2d --version
+    # # Inspect linkage
+    # - otool -L $PREFIX/bin/mp2d           # [osx]
+    # - conda inspect linkages --show-files --groupby=dependency mp2d
+    # - conda inspect objects -p $PREFIX $PKG_NAME   # [osx]
```

Comment:
These commands are generally not used/preferred anymore as the functionality is part of conda-build.
```suggestion
```